cooperatives/decidim

```diff
@@ -0,0 +1,25 @@
+---
+title: Decidim
+parent: Cooperatives
+---
+
+# Decidim
+
+|                   |                                          |
+|:------------------|:-----------------------------------------|
+| model             | Built Externally, Agencies Contribute
+| service type      | 
+| country           | Spain
```

Comment:
I think Decidim is being used outside of Spain as well. Can you please double check this on their website or with a quick internet search and update if so before we merge? @abbeyns149